### DISCUSSION TITLE
If Historical Weather Forecast Data are concatenated day-ahead weather forecasts?

### DISCUSSION THREAD

**Discussion by shouyisxty** [open-meteo/open-meteo#365] [Q&A]
If the historical weather forecasts data are concatenated one-day ahead daily weather forecasts?  

The “Forecast Days” setting has time horizons between 1 day to 16 days. When I set different Forecast Days, the downloaded historical weather forecasts are the same.  For example, using NOAA GFS given time between 5/1/2023 - 6/1/2023. So I assume the historical weather forecasts may be the concatenated one-day ahead daily weather forecasts. Is it right? Hope to get confirmed on this point. Thanks a lot!

**Comment by patrick-zippenfenig** [comment]
Almost. It is not concatenated one-day ahead forecast but 6 hours ahead forecasts for GFS.

![e589484a-d45d-44a8-9336-39384240ee03_1491x401](https://github.com/open-meteo/open-meteo/assets/11670456/0668af62-c9a9-48e0-92d5-312ed399bb82) See here: https://openmeteo.substack.com/i/42375179/chapter-model-runs

Typically the first hour of each weather model run is directly assembled from measurements and satellites. It is therefore close to "ground-truth" (with a gridding bias of 13 km case of GFS).


For North America and Europe, weather models with 1-2 km resolution that update every hour are available. Because of hourly updates, the "past data" available through the open-meteo API are very close to measurements.
